### PR TITLE
Routing CATs fixes for GH issues: #86 and #87

### PR DESCRIPTION
**Pull request by markstgodard**
This PR addresses GH issues both https://github.com/cloudfoundry/cf-acceptance-tests/issues/86 and https://github.com/cloudfoundry/cf-acceptance-tests/issues/87:

- Fix tests so they delete app properly

- Add missing app logging for session affinity tests

Note, talked to Amit and he was ok with a single PR for two issues, since the changes were in same area.

@markstgodard && @chavdarch

### CODE REVIEW

File: routing/routing_suite_test.go [hunk 1]
```diff
@@ -68,16 +68,17 @@ func StartApp(app string) {
 	Expect(cf.Cf("start", app).Wait(CF_PUSH_TIMEOUT)).To(Exit(0))
 }
 
-func PushApp(asset, buildpackName string) string {
-	app := PushAppNoStart(asset, buildpackName)
-	StartApp(app)
-	return app
+func PushApp(appName, asset, buildpackName string) {
+	PushAppNoStart(appName, asset, buildpackName)
```
Comment (robdimsdale) [inline routing/routing_suite_test.go]: We should be setting the backend here, between `push --no-start`, and `start`: see https://github.com/cloudfoundry/cf-acceptance-tests/blob/master/README.md#code-conventions

**Comment by robdimsdale** [comment]
@markstgodard && @chavdarch thanks for submitting this PR. We have a comment inline about explicitly setting the backend, and a more general observation that a lot of these helper methods are exported, but they're only used in the same package so maybe they don't need to be?

Let us know if you have any questions, thanks.

**Comment by ishustava** [comment]
Hi @robdimsdale,

Thank you for the feedback. We have fixed that. Please let us know if that works.

@atulkc and @ishustava

**Comment by APShirley** [comment]
Looks good, merging.

- Al & @nimakaviani

**Pull request merged**